test/e2e/csi/testsuites/testsuites.go

```diff
@@ -426,7 +428,7 @@ type TestDeployment struct {
 func NewTestDeployment(c clientset.Interface, ns *v1.Namespace, command string, pvc *v1.PersistentVolumeClaim, volumeName, mountPath string, readOnly bool) *TestDeployment {
 	generateName := "mayastor-volume-tester-"
 	selectorValue := fmt.Sprintf("%s%d", generateName, rand.Int())
-	replicas := int32(1)
+	replicas := int32(e2e_config.GetConfig().CSI.Replicas)
```

Comment:
Not related to this, but seeing int32 here is making me wonder if moac validates the spec when creating a volume. Could be another test, though could probably do it in the moac tests.